components/LessonCard.tsx

```diff
@@ -9,23 +9,38 @@ type Props = {
   lessonCount: number
   hourCount: string | number
   description: string
+  currentState?: string
 }
 
 const LessonCard: React.FC<Props> = props => {
   return (
     <div className="card shadow-sm mt-3">
       <div className="card-body p-2">
         <div className="row no-gutters">
+          {props.currentState === 'completed' && (
+            <span className="badge badge-pill text-white position-absolute lesson-card__container_completed">
```

Comment:
remove `text-white`, use `badge-success` instead. Documentation of badge-pill states: https://getbootstrap.com/docs/4.4/components/badge/#pill-badges